CompuCell3D/core/CompuCell3D/plugins/FocalPointPlasticity/FocalPointPlasticityPlugin.cpp

```diff
@@ -649,310 +636,241 @@ double FocalPointPlasticityPlugin::changeEnergy(const Point3D &pt,const CellG *n
 	Coordinates3D<float> centMassNewBefore;
 
 
-	if(oldCell){
-		centMassOldBefore.XRef()=oldCell->xCM/(float)oldCell->volume;
-		centMassOldBefore.YRef()=oldCell->yCM/(float)oldCell->volume;
-		centMassOldBefore.ZRef()=oldCell->zCM/(float)oldCell->volume;
+	if (oldCell) {
+		centMassOldBefore.XRef() = oldCell->xCM / (float)oldCell->volume;
```

Comment:
It woudl be great to fix this ugly syntax centMassOldBefore.YRef() = ... we do not have to do it now but I dont like it even though I wrote it ;)

Reply:
Implemented in https://github.com/CompuCell3D/CompuCell3D/pull/361/commits/66ddddbcbb07ec9f6d707e591dda1ecf0d7aeae3